src/app/containers/OnDemandHeading/index.jsx

```diff
@@ -74,14 +76,15 @@ const OnDemandHeadingContainer = ({
 OnDemandHeadingContainer.propTypes = {
   idAttr: string,
   brandTitle: string.isRequired,
-  releaseDateTimeStamp: number.isRequired,
+  releaseDateTimeStamp: number,
```

Comment:
I know this is still a draft but there should always be a releaseDateTimestamp. If you have found fixture data that does not contain the releaseDateTimestamp, it needs updated with the live Ares response